docs/source/architecture/worklock.rst

```diff
@@ -7,20 +7,22 @@ WorkLock
 Overview
 --------
 
-`WorkLock` is a novel, permissionless network node setup mechanism, developed at NuCypher, which requires participants
-to temporarily stake ETH and operate NuCypher network threshold cryptography nodes in order to be allocated NU, the native
+`WorkLock` is a novel, permissionless decentralized network node setup mechanism, developed at NuCypher, which requires
+participants to temporarily escrow ETH and operate NuCypher nodes in order to be allocated NU, the native
```

Comment:
"_novel, permissionless decentralized network node setup mechanism_" <- That's a tough sentence to read, particularly if you're not native. It has 5 adjectives in a row!